tests/backends/data/test_s3.py

```diff
@@ -603,18 +613,14 @@ def test_backends_data_s3_write_with_create_index_operation(
 
     assert list(
         filter(
-            lambda record: record[1] == logging.ERROR,
+            lambda record: record[1] in [logging.ERROR, logging.WARNING],
             caplog.record_tuples,
         )
     ) == (
         [
-            (
-                "ralph.backends.data.s3",
-                logging.ERROR,
-                f"Failed to encode JSON: {error}, for document {data[0]}",
-            )
+            ("ralph.backends.data.s3", logging.ERROR, msg),
+            ("ralph.backends.data.s3", logging.WARNING, msg),
```

Comment:
Should we log errors while  `ignore_errors == True` as `WARNING` ?

I don't have a strong opinion, but opening this in case someone does :)

Reply:
Good point) I think we could indeed decrease it) Don't have a strong opinion on it too)

Reply:
I don't have one either!